bin/hdfcoinc/pycbc_coinc_hdfinjfind

```diff
@@ -33,6 +33,7 @@ def keep_ind(times, start, end):
     time_sorting = times.argsort()
     times = times[time_sorting]
     indices = numpy.array([], dtype=numpy.uint32)
+    left = numpy.searchsorted(times, start, side='left')
```

Comment:
can you call 'left' and 'right' something else to avoid a context mixup again ..

Reply:
Ok, I've renamed them in the function.